### ISSUE TITLE
[29] Icon does not have text alternative

### ISSUE THREAD

**Issue by bjgordon** [AusDTO/dta-website#302]

**Comment by bjgordon** [comment]
Reprot to recruiterbox. Suggest they use the right character for the back arrow. e.g. `&larr;` (←)

**Comment by bjgordon** [comment]
Reported to recruiterbox. They fixed their high priority issue, but the remaining issues have gone to their Product Team for further consideration in future prioritization. Since this is a low severity issue, we won't track their progress, and I'll close this issue off.
